content/departments/product-engineering/engineering/code-graph/code-insights/processes.md

```diff
@@ -51,6 +51,14 @@ Our current process is as follows:
 
   - Teammates can **_reorder_ proposed issues** on the ["Next iteration" board](https://github.com/orgs/sourcegraph/projects/200/views/4) before the mid-iteration sync to their liking. The order at the time of the mid-iteration Monday sync is the _proposed order_.
 
+  - Often, planning work involves RFCs or designs that require engineering input. For this process:
+
+    - The goal of a product-manager-created RFC is to arrive at scoped outcomes for either designs or engineering: this is collected in a "Decisions for Design" or "Decisions for Engineering" section.
+    - At this point, the engineering project lead or designer becomes the driver for taking the work through its next stage. For engineering, this may optionally involve an engineering RFC or new implementation discussion among engineers on the existing RFC. For design, this usually involves figma mocks. From those stages, the respective project lead or designer and engineers can create github issues to put into proposed.
```

Comment:
```suggestion
    - At this point, the engineering project lead or designer becomes the driver for taking the work through its next stage. For engineering, this may optionally involve an engineering RFC or new implementation discussion among engineers on the existing RFC. For design, this usually involves Figma mocks. From those stages, the respective project lead or designer and engineers can create github issues to put into proposed.
```